recipes-kernel/linux/dts/add-vsc8531-ethernet.dts

```diff
@@ -0,0 +1,30 @@
+#include <dt-bindings/net/mscc-phy-vsc8531.h>
+
+&eth0 {
+        phy0: ethernet-phy@7 {
```

Comment:
while I approve this patch to get it into our device tree & working, we really should change this label to ` ethernet-phy@0` but when I tried it previously, I got some conflicts, there might need to be another change too. I've not looked into it yet.

Reply:
The number behind the `@` is the MDIO address